pyquil/quil.py

```diff
@@ -514,16 +514,14 @@ def wrap_in_numshots_loop(self, shots: int) -> "Program":
         self.num_shots = shots
         return self
 
-    def _out(self, allow_placeholders):
+    def _out(self):
```

Comment:
This method does not appear to be called anywhere, and is identical to `Program.out` if we remove the `allow_placeholders` arg.

It looks like `_out` was added in e134e3dd1a205afd5aab1a3577fdf63443a79649#diff-843483dd82a46bc0f8d1b3eac66ba00fR363, possibly intended to be a debugging aid? But AFAICT, the `allow_placeholders` arg was never added anywhere else.

Maybe we should just delete `_out` since it's a private unused method and apparently has always been broken.